fact/src/config.rs

```diff
@@ -20,4 +20,8 @@ pub struct FactConfig {
     /// Whether a small healthcheck probe should be run
     #[arg(long)]
     pub healthcheck: bool,
+
+    /// Whether to perform a pre flight check
+    #[arg(long, default_value_t = false)]
+    pub skip_preflight: bool,
```

Comment:
[nit] I think this would be more consistent with the rest of the code
```suggestion
    pub skip_pre_flight: bool,
```

If we want the argument on the executable to not have the second hyphen, we can change the live above to have `long = "skip-preflight"`